com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ProtoUtil.java

```diff
@@ -26,6 +26,15 @@
 public class ProtoUtil {
 
     public static void merge(IResource input, Builder builder) throws IOException, CompileExceptionError {
+        byte[] content = input.getContent();
```

Comment:
This did previously throw `Caused by: java.lang.NullPointerException: Cannot read the array length because "bytes" is null` which was really unhelpful. Now we check if the content of the resource is nil and throw a compile exception and reference the resource.